src/test/java/ItemListTest.java

```diff
@@ -0,0 +1,222 @@
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class ItemListTest {
+
+    @Test
```

Comment:
create @Before annotation to do the setup before executing any testcase.
You can move ItemList initialization in the setup method.
follow : 
https://junit.org/junit4/javadoc/4.12/org/junit/Before.html